docs/resources/cluster.md

```diff
@@ -2,21 +2,68 @@
 page_title: "ybm_cluster Resource - YugabyteDB Managed"
 description: |-
   The resource to create a YugabyteDB cluster. Use this resource to create both
-          single- and multi-region clusters. You can also use this resource to bind allow lists to the cluster
-          being created; restore previously taken backups to the cluster being created;
-          and modify the backup schedule of the cluster being created.
+  single- and multi-region clusters. You can also use this resource to bind allow lists to the cluster
+  being created; restore previously taken backups to the cluster being created;
+  and modify the backup schedule of the cluster being created.
 ---
 
 # ybm_cluster (Resource)
 
 The resource to create a YugabyteDB cluster. Use this resource to create both 
-		single- and multi-region clusters. You can also use this resource to bind allow lists to the cluster 
-		being created; restore previously taken backups to the cluster being created; 
-		and modify the backup schedule of the cluster being created.
+single- and multi-region clusters. You can also use this resource to bind allow lists to the cluster 
+being created; restore previously taken backups to the cluster being created; 
+and modify the backup schedule of the cluster being created.
 
 
 ## Example Usage
 
+To create a single region cluster by using common credentials for both YSQL and YCQL API
+
+```terraform
+variable "password" {
+  type        = string
+  description = "YSQL and YCQL Password."
+  sensitive   = true
+}
+
+# Single Region Cluster
+resource "ybm_cluster" "single_region_cluster" {
+  cluster_name = "single-region-cluster"
+  cloud_type   = "GCP"
+  cluster_type = "SYNCHRONOUS"
+  cluster_region_info = [
+    {
+      region    = "us-west1"
+      num_nodes = 3
+      vpc_id    = "example-vpc-id"  #Optional
+    }
+  ]
+  cluster_tier           = "PAID"
+  cluster_allow_list_ids = ["example-allow-list-id-1", "example-allow-list-id-2"] #Optional
+  fault_tolerance        = "ZONE"
+  node_config = {
+    num_cores    = 2
+    disk_size_gb = 50  #Optional
+  }
+  backup_schedules = [
+    {
+      state                    = "ACTIVE"
+      retention_period_in_days = 10
+      time_interval_in_days    = 10
+    }
+  ]  #Optional
+  credentials = {
+    ysql_username = "example_ysql_user"
+    ysql_password = var.ysql_password
```

Comment:
This example uses separate passwords for YSQL and YCQL, but the text above says common credentials.